src/autosuggest/interfaces.ts

```diff
@@ -80,6 +80,11 @@ export interface AutosuggestProps
    */
   enteredTextLabel?: AutosuggestProps.EnteredTextLabel;
 
+  /**
+   * Specifies whether to display the custom value indicator (for example, `Use "${value}"`).
+   */
+  hideEnteredTextLabel?: boolean;
```

Comment:
Do we really need a new property? Or could we just hide the entered text option if `enteredTextLabel` is undefined/empty ?

Reply:
@timogasda wouldn't that be less backwards compatible, as it would impact the existing default? Additionally, we expect the majority of autosuggest customers to continue with the current default, wheras this property would be a smaller optional subset of use cases.